app/app.js

```diff
@@ -1,14 +1,25 @@
 var app = angular.module('bucketlistApp',['ngRoute']);
 
-app.config(function($routeProvider){
-  $routeProvider.when('/', {
+app.config(['$routeProvider', '$httpProvider', function($routeProvider, $httpProvider){
+  $routeProvider
```

Comment:
I prefer using ngRoute, I've only used UI-Router once. I will take a look at it again